source/overview/faq.rst

```diff
@@ -0,0 +1,37 @@
+Frequently Asked Questions 
+==========================
+
+General Questions 
+-----------------
+
+Why was Mattermost created?
+    Mattermost was created to offer an alternative to propreitary SaaS services. For more info, please go to mattermost.org and click "Why we made Mattermost".
+
```

Comment:
Mattermost.org is not available for now. See https://pre-release.mattermost.com/core/pl/wnwfwp11i3b9icrtpdsqjjtfgc

You could use a direct link to https://www.mattermost.org/why-we-made-mattermost-an-open-source-slack-alternative/ but there's no knowing how long that link will last.

Maybe a paragraph or two summary of the "Why we made Mattermost" page?